note-app/src/Component/Main/Main.css

```diff
@@ -37,5 +36,36 @@
 .editorNormal-btn:hover{
     background-color: var(--primary-color);
 }
+.notePallet-container{
+    background-color: #fff;
+  box-shadow: 2px 2px 2px 2px var(--boxShadow-color);
+  position: absolute;
+  top: 42%;
+  left: 44.5%;
+  justify-content: space-between;
+  gap: .5rem;
+  border-radius: 5px;
+}
+.note-color{
+    width: 2rem;
+  height: 2rem;
+  border: 1px solid var(--secondary-color);
+  border-radius: 50%;
+  cursor: pointer;
+}
+.note-color-pink{
+    background-color: #f28b82;
```

Comment:
here you can also use to put value as variable

Reply:
will do that